AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs

```diff
@@ -165,13 +165,23 @@ public void ScrollToTheTopOfThePage()
             ScrollTo(By.TagName("header"));
         }
 
-        public void ClickToProgress(By element, By oldElement, int timeout = 20)
+        public void ClickToProgress(By element, By newElement, int timeout = 20)
         {
             for (var i = 0; i < timeout; i++)
             {
-                JavascriptClick(element);
-                if (Driver.FindElement(oldElement) == null) return;
-                Thread.Sleep(TimeSpan.FromSeconds(1));
+                try 
+                {
+                   Console.WriteLine($"loop {i}");
+                    JavascriptClick(element);
+                    Thread.Sleep(TimeSpan.FromSeconds(1)); 
+                    Driver.WaitUntilVisible(newElement, 1);
```

Comment:
I don't think both wait times are required. Wouldn't **Driver.WaitUntilVisible(newElement, 1);** do the trick?

Reply:
Fixed, I had a reason for doing it this way but maybe a wait will work.